Bluepill-cli/Bluepill-cli/Bluepill/Bluepill.m

```diff
@@ -480,16 +544,38 @@ - (void)finishWithContext:(BPExecutionContext *)context {
             context.finalExitStatus = BPExitStatusAppCrashed;
             NEXT([self proceed]);
             return;
+        case BPExitStatusSimulatorDeleted:
+            self.finalExitStatus = BPExitStatusSimulatorDeleted;
+            self.exitLoop = YES;
+            return;
     }
 
 }
 
+- (void)writeDeviceIDFile {
```

Comment:
Thought about using other IPC mechanism. If there are continuous information exchange, other IPC would be preferred.   but since here we just want to pass a simple string, one time thing, other IPC mechanism seems overkill and make code more complicated.  Writing & reading a temp file are very easy with only a few lines code. If the driver and bp will exchange more information in the future, we can switch to other IPC. 
Oscar & Ashit, What do you think?